src/main/transports/electron-offline-net.ts

```diff
@@ -38,7 +38,7 @@ export class ElectronOfflineNetTransport extends ElectronNetTransport {
         this._requestSuccess();
         return response;
       } catch (error) {
-        if (error instanceof HTTPError && error.status != 'rate_limit') {
+        if (error instanceof HTTPError && error.status !== 'rate_limit') {
```

Comment:
Isn't https://eslint.org/docs/rules/eqeqeq turned on?

Reply:
Is this not part of the Sentry eslint config that we extend?

Reply:
TIL `eqeqeq` is not part of `eslint:recommended` so we do not have it turned on! Let me open a PR on the monorepo and chat there.